src/components/Form/Form.js

```diff
@@ -0,0 +1,54 @@
+import "./Form.style.css";
+import Button from "../Button/Button.js";
+
+export default function Form({ onAddActivity }) {
+	function handleSubmit(e) {
+		e.preventDefault();
+		const formData = new FormData(e.target);
+		const data = Object.fromEntries(formData);
+		if (data.isForGoodWeather) {
+			data.isForGoodWeather = true;
+		} else {
+			data.isForGoodWeather = false;
+		}
+		e.target.reset();
+		e.target.name.focus();
```

Comment:
See above if you keep this variant:

```suggestion
		event.target.reset();
		event.target.name.focus();
```